api/wasm/v0/cpp/proxy_wasm_externs.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Proxy-WASM ABI.
+ */
+// NOLINT(namespace-envoy)
+
+#pragma once
+
+#include <stddef.h>
+#include <stdint.h>
+
+//
+//  ABI calls from the host into the VM.
```

Comment:
nit: extra leading space on this and next comment block.

Reply:
Done